portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -45,19 +43,21 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   }
 
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String userComment = request.getParameter("comment-input");
-    long timestamp = System.currentTimeMillis();
+    String name = request.getParameter("name-input");
+    String content = request.getParameter("comment-input");
+    long time = System.currentTimeMillis();
 
     // Uses a hidden form to get the page the user sent the request from
     // Allows redirect to work correctly even if comments section is present on multiple pages
     String pageToRedirect = request.getParameter("page-name");
     
     Entity commentEntity = new Entity("Comment");
-    commentEntity.setProperty("text", userComment);
-    commentEntity.setProperty("time", timestamp);
-    
+    commentEntity.setProperty("name", name);
+    commentEntity.setProperty("content", content);
+    commentEntity.setProperty("time", time);
+
     // TODO remove once code is able to read from database
```

Comment:
Is this TODO still relevant?

Reply:
Unfortunately yes. My next pull request should add the code to actually read from the database.